common/changes/nickpape-rush-pinned-versions_2017-02-24-00-38.json

```diff
@@ -0,0 +1,15 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush-lib",
+      "comment": "Add a \"pinnedVersions\" option to rush.json, which will add dependencies to the top-level package.json. These dependencies are resolved first, and thus this mechanism can be used to control versions of second-level dependencies.",
+      "type": "minor"
+    },
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add a \"pinnedVersions\" option to rush.json, which will add dependencies to the top-level package.json. These dependencies are resolved first, and thus this mechanism can be used to control versions of second-level dependencies.",
```

Comment:
Suggest to change to:

"Add a \"pinnedVersions\" option to rush.json, which will add dependencies to the common package.json. Since these dependencies are installed first, this mechanism can be used to control versions of unconstrained second-level dependencies."